arch/Kconfig

```diff
@@ -477,6 +477,15 @@ config GEN_ISR_TABLES
 	  script to generate its interrupt tables. This mechanism will create
 	  an appropriate hardware vector table and/or software IRQ table.
 
+config GEN_ISR_TABLES_SCIRPT
```

Comment:
```suggestion
config GEN_ISR_TABLES_SCRIPT
```

Reply:
This was a stupid mistake.
Thank you for pointing it out. It has been addressed.